components/site/src/lib.rs

```diff
@@ -591,7 +591,14 @@ impl Site {
         }
         // We're fine with missing static folders
         if self.static_path.exists() {
-            copy_directory(&self.static_path, &self.output_path, self.config.hard_link_static)?;
+            if let Some(gs) = &self.config.ignored_static_globset {
+                copy_directory_with_ignore_globset(
+                    &self.static_path,
+                    &self.output_path,
+                    self.config.hard_link_static,
+                    gs,
+                )?;
+            }
```

Comment:
We would never copy the static folder if the user hasn't set ignored_static from what I'm understanding?

Reply:
:facepalm: fixing now